Spec/Auth.swift

```diff
@@ -1717,14 +1718,12 @@ class Auth : QuickSpec {
                     waitUntil(timeout: testTimeout) { done in
                         auth.authorise(nil, options: authOptions) { tokenDetails, error in
                             expect(authCallbackHasBeenInvoked).to(beTrue())
-
-                            authCallbackHasBeenInvoked = false
```

Comment:
> I guess you mean + subsequent requestwithany arguments here?

No, we need to test that the defaults are used for subsequent authorisations so if we passed in arguments then we'd be changing those defaults?

>> Remove force completely

> Remove it only from tests target, or also from the lib target?

Not sure what you mean by target, but it should be removed as this option is no longer available as part of the tests you are implementing